tablesnap

```diff
@@ -56,7 +56,8 @@ default_chunk_size = 256
 default_listen_event = 'IN_MOVED_TO'
 
 class UploadHandler(pyinotify.ProcessEvent):
-    def my_init(self, threads=None, key=None, secret=None, bucket_name=None,
+    def my_init(self, threads=None, key=None, secret=None, token=None,
```

Comment:
Right? Do you know what this is? It doesn't seem to be called explicitly anywhere. Is it some implicit thing?